src/preview.ts

```diff
@@ -106,6 +176,12 @@ export async function activate(
   );
   context.subscriptions.push(disposable);
 
+  disposable = vscode.commands.registerCommand(
+    'codeart-binexplore.previewBinary',
+    previewOutput
+  );
+  context.subscriptions.push(disposable);
```

Comment:
nit: no need to use `disposable` variable. I believe that doing something like this makes code easier to read:
```suggestion
  context.subscriptions.push(
    vscode.workspace.onDidOpenTextDocument(analyzeOpenedDocument),
    vscode.commands.registerCommand(
      'codeart-binexplore.previewBinary',
      previewOutput
    ),
    vscode.workspace.registerTextDocumentContentProvider(
      EXTENSION_SCHEME,
      provider
    ),
    vscode.languages.registerDocumentSymbolProvider(
      {
        scheme: EXTENSION_SCHEME,
      },
      new CodeArtSymbolProvider()
    )
  );
```

Reply:
I am ok with both, do not have specific preference. It would be good to standardize the development, so let's always go in the future with your recommended style.